patterns/nvidia-gpu-efa/README.md

```diff
@@ -17,46 +17,52 @@ The following components are demonstrated in this pattern:
 
 ## Code
 
-```terraform hl_lines="24-26 32-67"
-{% include  "../../patterns/nvidia-gpu-efa/eks.tf" %}
-```
+The code consists of the following files:
```

Comment:
please revert these changes - what is being provided is described above, this section is highlighting those key points for this pattern through the code (per the section title)